doc/crypto/api/library/status.rst

```diff
@@ -55,6 +55,16 @@ Some of the common status codes have a more precise meaning when returned by a f
     * - Error code
       - Meaning in the |API|
 
+    * - :code:`PSA_ERROR_NOT_SUPPORTED`
+      - `[PSA-STAT]` recommends the use of :code:`PSA_ERROR_INVALID_ARGUMENT` for invalid parameter values.
+
+        In the |API|, this is relaxed for algorithm identifier and key parameters, when either :code:`PSA_ERROR_INVALID_ARGUMENT` or :code:`PSA_ERROR_NOT_SUPPORTED` can be returned.
+
+    * - :code:`PSA_ERROR_INVALID_ARGUMENT`
+      - `[PSA-STAT]` recommends the use of :code:`PSA_ERROR_NOT_SUPPORTED` for unsupported parameter values.
+
+        In the |API|, either :code:`PSA_ERROR_INVALID_ARGUMENT` or :code:`PSA_ERROR_NOT_SUPPORTED` can be returned when unsupported algorithm identifier or key parameters are used.
```

Comment:
As above:

```suggestion
        In the |API|, either :code:`PSA_ERROR_INVALID_ARGUMENT` or :code:`PSA_ERROR_NOT_SUPPORTED` can be returned when unsupported algorithm identifier or key type parameters are used. This allows implementations to avoid having to recognize all the cryptographic mechanisms that are defined in the PSA specification but not provided by that particular implementation.
```